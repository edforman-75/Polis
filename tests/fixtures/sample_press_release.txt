Spanberger Hosts Jobs & Innovation Rally in Richmond
2024-10-12 â€” Richmond, VA

Today, Abigail Spanberger hosted a grassroots rally focused on good jobs, small business growth, and innovation in Central Virginia. Local union leaders and entrepreneurs joined to share how new investments are helping families.

Doors open at 2:15 PM; ASL provided; media check-in on site.
