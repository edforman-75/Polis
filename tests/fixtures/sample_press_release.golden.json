{
  "doc_id": "SAMPLE-SPANBERGER-0001",
  "title": "Spanberger Hosts Jobs & Innovation Rally in Richmond",
  "date": "2024-10-12",
  "source": "fixture://sample_press_release",
  "primary_label_id": "ANN.EVT.RALLY",
  "score": 5.0
}
